apps/system/js/bluetooth_transfer.js

```diff
@@ -643,6 +645,33 @@ var BluetoothTransfer = {
     var body = {id: 'unknownMediaTypeToOpenFile', args: {fileName: fileName}};
     CustomDialog.show('cannotOpenFile', body, confirm, null, screen)
     .setAttribute('data-z-index-level', 'system-dialog');
-  }
+  },
 
+  _cleanupTempFileInDir: function bt__cleanupTempFileInDir(evt) {
```

Comment:
This function is declared to take an evt, but you're passing evt.transferInfo to it, which is confusing.